pkg/checks/traceroute/check.go

```diff
@@ -32,6 +41,7 @@ func NewCheck() checks.Check {
 		config:     Config{},
 		traceroute: TraceRoute,
 		metrics:    newMetrics(),
+		tracer:     otel.Tracer("tracer.traceroute"),
```

Comment:
let's use the `CheckName` here? I don't think there will be any conflicts with other traces, as each check will have the one.